src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -86,6 +89,7 @@ const MapOverlays = ({
                 }
                 onPress={() => {
                   onBuildingTap(building.id);
+                  setBuildingMarkerLocation(building);
```

Comment:
Might be better to fuse these two methods into a new method called `onBuildingTap(Building)`. That was its a bit clearer what even the function is handling, and there no need to call two methods.

Reply:
`onBuildingTap` sets the `setShowBuildingInfo` to true which works on, so that we see the sliding panel, whereas in `setBuildingMarkerLocation` only sets the building location in the omnibox. If we were to combine both, it would lead us to seeing the sliding panel when in the omnibox display view.

Reply:
Not necessarily, although you are right to question what effect this might have on the app. Actually the current way that onPress() is set up with those two methods _**can**_ lead to some nasty bugs since it would call `onBuildingTap` and do:
```tsx
    setShowBuildingInfo(true);
    setTappedBuilding(tappedBuilding);
```

even when planning or travelling. Which isn't what we want

From what I can see both methods are often called together (except once):
![image](https://user-images.githubusercontent.com/23544999/78418316-178b2680-7609-11ea-858f-169a9c98901a.png)
![image](https://user-images.githubusercontent.com/23544999/78418317-1bb74400-7609-11ea-8f79-87ff8161db6b.png)
![image](https://user-images.githubusercontent.com/23544999/78418321-21148e80-7609-11ea-9b60-ea09a780dd04.png)

Again this is minor, but it would make it simpler to have one method for when a building is pressed. This method would act as such:
```tsx
  const onBuildingTap = (building: Building) => {
    if (travelState === TravelState.NONE) {
      setShowBuildingInfo(true);
      setTappedBuilding(building.id);
    } else if (travelState === TravelState.PLANNING) {
      if (endLocationFocused) {
        setEndLocation(building);
      } else {
        setStartLocation(building);
      }
    }
  };
```

I've tested it out locally and it seems to work just like before! Lemme know if you run into trouble @r-saba @cindyslittleplanet, I can come help.

Reply:
I agree with @TheGreatMarkus, these event handlers should not call other handlers to avoid any unwanted behaviors

Reply:
I also agree with @TheGreatMarkus to put the `setBuildingMarkerLocation(building)` within the `setShowBuildingInfo` for clarity.

Reply:
Done!